### DISCUSSION TITLE
Where can I download the Enterprise and Pro `.gemspec` files?

### DISCUSSION THREAD

**Discussion by lorrin** [sidekiq/sidekiq#6323] [General]
We vendor the Sidekiq Enterprise and Pro gems in our source tree under `vendor/gems/sidekiq-(ent-pro)-<version>/`. (And we add them as path based dependencies in our own Gemfile). Dependabot is cranky about this (_"To use path-based dependencies with Dependabot the paths must be relative, resolve to a directory in this project's source code, and contain a valid Ruby project."_). It turns out the "valid Ruby project" bit is caused by the lack of a `.gemspec` file. For now, we've worked around this by adapting https://github.com/sidekiq/sidekiq/blob/main/sidekiq.gemspec by hand, but it would be nice to have the official ones.

**Comment by mperham** [comment]
You download them as part of the .gem file itself.

**Reply by mperham** [reply]
I.e. the gemspec is in the gem.

**Reply by lorrin** [reply]
Oh, interesting. I was using `gem unpack` to unpack the code from the gem and didn't realize there was more in there. I see now `gem specification` will get me a YAML rendition of the Gemspec, but I'm still unclear how to get the actual `.gemspec` file.

**Reply by mperham** [reply]
Me too or I'd tell you!

**Reply by mperham** [reply]
Seems like this would be a good reason to open a rubygems issue: "gem unpack doesn't work with Dependabot because ..."
